src/domains/issue/controller/issue.controller.ts

```diff
@@ -0,0 +1,21 @@
+import { type Request, type Response, Router } from 'express';
+import { type IssueService, IssueServiceImpl } from '@domains/issue/service';
+import { db, validateRequest } from '@utils';
+import { type IssueRepository, IssueRepositoryImpl } from '@domains/issue/repository';
+import HttpStatus from 'http-status';
+import { type EventRepository, EventRepositoryImpl } from '@domains/event/repository';
+import { IssuePauseParams } from '@domains/issue/dto';
+
+export const issueRouter = Router();
+
+const issueRepo: IssueRepository = new IssueRepositoryImpl(db);
+const eventRepo: EventRepository = new EventRepositoryImpl(db);
+const issueService: IssueService = new IssueServiceImpl(issueRepo, eventRepo);
+
```

Comment:
May change id to issueId

Reply:
Solved according to established naming convention.